app/components/questions/TrafficIntensity.js

```diff
@@ -0,0 +1,176 @@
+import { useEffect, useState } from 'react'
+import useFetch from 'use-http'
+import {
+  ResponsiveContainer,
+  PieChart,
+  Pie,
+  Cell,
+  Tooltip,
+  Legend,
+} from 'recharts'
+import { FailureNotification } from '../../components/Notification'
+import { Color } from '../../site/theme'
+import { Formatter } from '../../site/metrics'
+
+const ENDPOINT = `${process.env.NEXT_PUBLIC_API}/community/metrics`
+
+const CITY_PART_COLOR = {
+  "Central": "#332288",
+  "Far North Side": "#1978AD",
+  "Far Southeast Side": "#117733",
+  "Far Southwest Side": "#44AA99",
+  "North Side": "#88CCEE",
+  "Northwest Side": "#DCA139",
+  "South Side": "#CC6677",
+  "Southwest Side": "#AA4499",
+  "West Side": "#882255",
+};
+
+const EMPTY_MOST_TRIPS = {
+  name: "?",
+  totalTripsFormatted: "?",
+};
+const AREA_COLORS = [
+  Color.Forest,
+  Color.Gold,
+  Color.Magenta,
+  Color.Cerulean,
+  Color.HealthPink,
+  Color.Indigo,
+  Color.Red,
+  Color.Sky,
+  Color.Salmon,
+  Color.Maroon,
+  ]
+
+function transformData(response, error, selectedYear) {
+  if (!response || error) {
+    return {
+      chartData: [],
+     
+    }
+  }
+  
+  var traffic_intensity_year = ((selectedYear === 2020) ? 'traffic_intensity_2020'
+                  :(selectedYear === 2019) ? 'traffic_intensity_2019'
+                  :(selectedYear === 2018) ? 'traffic_intensity_2018'
+                  :(selectedYear === 2017) ? 'traffic_intensity_2017'
+                  :(selectedYear === 2016) ? 'traffic_intensity_2016'
+                  : 'Error');
+
+  const dataWithName = response.metrics.map(d => ({
+    // Get all fields from original record
+    ...d,
+   
+    // Add formatted data
+    traffic_intensity_formatted: parseFloat(Formatter.numberWithTwoDecimals(d[traffic_intensity_year])),
```

Comment:
You can remove the parse float. The parse float undoes the formatting.

Reply:
Yea we initially had `Formatter.numberWithTwoDecimals(d[traffic_intensity_year])` but it returned a string for `traffic_intensity_formatted`'s value which the Piechart did not like

Reply:
The formatted field should be for displaying the values outside of the chart. For the chart data, you can use the raw value `d[traffic_intensity_year]`, right? Maybe add a field:

```
{
  ...
  value: d[traffic_intensity_year],
  ...
}
```

Reply:
ahh I see. Yea the initial issue which led use to writing this was that there were too many significant figures for values for our dataset which led us to use the Formatter. Do you think it would be better to `ROUND` on the backend within the SQL query? @vingkan

Reply:
Oh, I see. What you did with the round should work, but there could still be situations where you get trailing decimals when the response is received by the frontend.

Ultimately, what we'd want to do is specify to the Recharts visualization how to format the data it displays. I don't know off the top of my head what it is, but if you look at the documentation or examples online you can find out how to pass a formatter function to it.

But that's not super important here, so let's merge the solution you and Shahzia did that is working and move on to adding Urdu support in the next PR.